google/cloud/security/iam/inventory/storage.py

```diff
@@ -454,6 +468,30 @@ def _open(self, existing_id):
                 [InventoryState.SUCCESS, InventoryState.PARTIAL_SUCCESS]))
             .one())
 
+    def _get_resource_rows(self, key):
+        """ Get the rows in the database for a certain resource
+
+        Args:
+            key (str): The key of the resource
+
+        Returns:
+            Object: The inventory db rows of the resource, 
+            IAM policy and GCS policy.
+
+        Raises:
+            Exception: if there is no such row or more than one.
+        """
+
+        qry = (self.session.query(Inventory)
```

Comment:
recommendation for different indentation, push self.session.query(Inventory) into the next line and then have .filter start in the same column so that qry is least indented and it's clear that this is a multiline statement.